### PR TITLE
feat: Add improved Konnect Analytics summary dashboard changelog

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

- I added a Konnect changelog entry to document the improved analytics summary dashboard.
 

DOCU-3585

### Testing instructions

Preview link: https://deploy-preview-6811--kongdocs.netlify.app/konnect/updates/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
❗ For reviewers: I added the "do not merge" label, but I'm wondering if that can be removed since `dev` was released? I'm not sure how `dev` will work for Konnect.

**Comment by Guaris** [comment]
@cloudjumpercat could you update the changelog to include this screenshot? 
summary-dashboard

**Review by Guaris — APPROVED** [review]

**Comment by cloudjumpercat** [comment]
Hi @sichvoge ! Just wanted to let you know that the Analytics summary dashboard changelog is ready to merge once the feature is released! Here’s a link to the preview of the changelog: https://deploy-preview-6811--kongdocs.netlify.app/konnect/updates/ 

Let me know if you have any questions!

File: app/konnect/updates.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## February 2024
+
+**Improved Analytics summary dashboard**
+: A new version of the Analytics summary dashboard is now available for all {{site.konnect_short_name}} users. This new version displays all analytics information on one page, which allows you to focus on your most critical API usage data. Additionally, you can now apply filters to tailor the summary dashboard to the analytics that matter the most to you.
```
Comment (sichvoge) [inline app/konnect/updates.md:15]:
```suggestion
: A new version of the Analytics summary dashboard is now available for all {{site.konnect_short_name}} users. This new version displays all analytics information on one page, which allows you to focus on your most critical API usage data.
```

Comment (sichvoge) [reply app/konnect/updates.md:15]: We will not support filtering at this phase. Also updated the Aha ticket.

**Review by sichvoge — DISMISSED** [review]
Thanks @cloudjumpercat. We should also update https://docs.konghq.com/konnect/analytics/#summary-dashboard-and-custom-reports to match the content of the new dashboard layout.

**Comment by cloudjumpercat** [comment]
> Thanks @cloudjumpercat. We should also update https://docs.konghq.com/konnect/analytics/#summary-dashboard-and-custom-reports to match the content of the new dashboard layout.

@sichvoge Thanks for that feedback! Angel said he'd create a new Jira ticket to cover that work for this feature, and one of us will pick that up. I can get back to updating the page you linked once I've wrapped up some work for the growth project that has a deadline coming up.

**Comment by cloudjumpercat** [comment]
I manually added this to #6961 (since there were so many commits, rebase, etc. I figured manually was easier). So I'll close this PR.

**Pull request closed without merging**